### DISCUSSION TITLE
Tasks max_active_tis_per_dag property not working in Deferrable Operators

### DISCUSSION THREAD

**Discussion by maguerreberry** [apache/airflow#29151] [General]
Hi everybody,

I realize that the property `max_active_tis_per_dag` is not taken into account in deferrable operators.

I'm using a Deferreable Operator, with `max_active_tis_per_dag=1`. When running multiple dag runs the task instance concurrency limit is ignore and multiple task instances are scheduled.

Is this a bug?

**Comment by hussein-awala** [comment]
What do you mean by the task instance concurrency limit is ignore? is it ignored for all the tasks or just for the deferred tasks?
It's normal to have multiple task in deferred state where in this state, the task pauses its execution, releases the worker slot, and submit a trigger to be picked up by the `Triggerer` process.

When you set `max_active_tis_per_dag`, Airflow ensures that (the count of running and queued tasks) <=  `max_active_tis_per_dag`, is it the case for you?
